cmd/go-filecoin/message.go

```diff
@@ -277,11 +277,6 @@ var msgStatusCmd = &cmds.Command{
 			}
 		}
 
-		// Look on chain
```

Comment:
If we don't want this in the message status command anymore we should remove the `OnChain` bool from the status reporting type.